templates/gene.html

```diff
@@ -333,46 +333,59 @@ <h4 class="modal-title" id="myModalLabel"></h4>
         renderer: '/static/bower_components/graph-viz-d3-js/dist/renderer'
     }
 });
-var unrelated_flag = false,
-    hpo_terms = {{ hpo_terms_json|safe }},
-    patients_status = {{ patients_status|safe }},
-    hom = {{ dot_hom_comp|safe }},
-    het = {{ dot_het|safe }};
-// draw hom first. can't draw two on the same page
-draw_hpo(hom, 'hom_comp', 0.01, 15, false, true);
+var patients_status = {{ patients_status|safe }},
+    options = {
+        mode: 'r', // r for recessive, d for dominant
+        exclude_hom: false,
+        unrelated_flag: true,
+        hpo_terms: {{ hpo_terms_json|safe }},
+        remove_bad_vars: true,
+        cutoffs: {
```

Comment:
what do u think of moving reading these cutoff values directly from the sliders in `transformData()` (and start at these value on the slider)?